lib/cli/defaults.json

```diff
@@ -8,7 +8,15 @@
     }
   },
   "backendless": {
-    "msgBroker": {}
+    "msgBroker": {},
+    "publicHost": "localhost",
```

Comment:
I am not sure about these defaults `publicHost| publicPort|publicProtocol`, for example, there is might not be `publicPort` and as a result, it will be a wrong `publicApiUrl` `https://api.backendless.com:9000`

Reply:
Removed from defaults